#
# old_revision [3189858dee662e616a3978601d74269683a55f48]
#
# patch "file1"
#  from [a43dc27b1c92cccc533ceb3a27035128e26e5b07]
#    to [8aa9b19b304a24ebcd71cfc4a27efc95f8ad8e11]
#
============================================================
--- file1	a43dc27b1c92cccc533ceb3a27035128e26e5b07
+++ file1	8aa9b19b304a24ebcd71cfc4a27efc95f8ad8e11
@@ -1,3 +1,4 @@ quux
 foo
 bar
 quux
+baz
